<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>281</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 87 16pt &quot;Arial Black&quot;;
font: 24pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Secretary</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="nextSec">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>410</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>20</pointsize>
      <weight>10</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 87 20pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>NEXT</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>480</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 254, 226);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>771</width>
       <height>324</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="sec1">
        <property name="font">
         <font>
          <family>Arial Black</family>
          <pointsize>36</pointsize>
          <weight>10</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 118, 26);
background-color: rgb(255, 2, 18);
selection-color: rgb(255, 0, 12);
selection-background-color: rgb(255, 0, 4);
font: 87 36pt &quot;Arial Black&quot;;</string>
        </property>
        <property name="text">
         <string>CAGAPE, BRANDON SAGE</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sec2">
        <property name="font">
         <font>
          <family>Arial Black</family>
          <pointsize>36</pointsize>
          <weight>10</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 118, 26);
background-color: rgb(255, 2, 18);
selection-color: rgb(255, 0, 12);
selection-background-color: rgb(255, 0, 4);
font: 87 36pt &quot;Arial Black&quot;;</string>
        </property>
        <property name="text">
         <string>GENABE, CHRIST JOHN</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sec3">
        <property name="font">
         <font>
          <family>Arial Black</family>
          <pointsize>36</pointsize>
          <weight>10</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 118, 26);
background-color: rgb(255, 2, 18);
selection-color: rgb(255, 0, 12);
selection-background-color: rgb(255, 0, 4);
font: 87 36pt &quot;Arial Black&quot;;</string>
        </property>
        <property name="text">
         <string>KIM, JOSEPH</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sec4">
        <property name="font">
         <font>
          <family>Arial Black</family>
          <pointsize>36</pointsize>
          <weight>10</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 118, 26);
background-color: rgb(255, 2, 18);
selection-color: rgb(255, 0, 12);
selection-background-color: rgb(255, 0, 4);
font: 87 36pt &quot;Arial Black&quot;;</string>
        </property>
        <property name="text">
         <string>ABSTAIN</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="backSec">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>410</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>20</pointsize>
      <weight>10</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 87 20pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <zorder>frame</zorder>
   <zorder>label</zorder>
   <zorder>nextSec</zorder>
   <zorder>backSec</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
